widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.kt

```diff
@@ -517,7 +517,9 @@ class ChatView(context: Context, attrs: AttributeSet?, defStyleAttr: Int, defSty
             theme = theme,
             companyName = companyName,
             positiveButtonClickListener = { onEngagementConfirmationDialogAllowed() },
-            negativeButtonClickListener = { onEngagementConfirmationDialogDismissed() }
+            negativeButtonClickListener = { onEngagementConfirmationDialogDismissed() },
+            link1ClickListener = {}, //TODO: will be implemented on the next task
```

Comment:
Can we maybe make this a little more specific? We are expecting these to be ToS and PP right? Maybe we call the listeners accordingly? Would add a lot of useful context to reading the code

Reply:
Actually, these are links one and two. The integrators can use them as they want.
Maybe it makes sense to change it to:
`linkClickListener = { url: String }`